moveit/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
kinetic: >= 3.0.2
melodic: >= 3.7.2

Reply:
Wow, good eyes you got there!
You are right, but these are meta packages and the exact content of the file sadly matters:

https://github.com/ros-planning/moveit/pull/1620